src/core/cmap.js

```diff
@@ -303,13 +304,28 @@ var CMap = (function CMapClosure() {
       }
       out.charcode = 0;
       out.length = 1;
+    },
+
+    get isIdentityCMap() {
+      if (!(this.name === 'Identity-H' || this.name === 'Identity-V')) {
+        return false;
+      }
+      if (this._map.length !== 0x10000) {
+        return false;
+      }
+      for (var i = 0; i <= 0xFFFF; i++) {
```

Comment:
Tiny nit: it's a bit strange to use 0x1000 above but then use 0xFFFF here. Can we just make it < 0xFFFF?